app/models/appeals/AppealSubmission.scala

```diff
@@ -431,90 +457,110 @@ object AppealSubmission {
     reasonableExcuse match {
       case "bereavement" => {
         AppealSubmission(
-          submittedBy = if (userRequest.isAgent) "agent" else "client",
-          reasonableExcuse = reasonableExcuse,
-          honestyDeclaration = userRequest.session.get(SessionKeys.hasConfirmedDeclaration).get == "true",
+          sourceSystem = "MDTP",
+          taxRegime = "VAT",
+          customerReferenceNo = s"VRN${userRequest.vrn}",
+          dateOfAppeal = LocalDateTime.now(),
+          isLPP = !userRequest.session.get(SessionKeys.appealType).contains(PenaltyTypeEnum.Late_Submission.toString),
+          appealSubmittedBy = if (userRequest.isAgent) "agent" else "client",
           agentDetails = agentDetails,
           appealInformation = BereavementAppealInformation(
-            `type` = "bereavement",
+            reasonableExcuse = reasonableExcuse,
+            honestyDeclaration = userRequest.session.get(SessionKeys.hasConfirmedDeclaration).get == "true",
             dateOfEvent = userRequest.session.get(SessionKeys.whenDidThePersonDie).get,
             statement = None,
             lateAppeal = isLateAppeal,
             lateAppealReason = userRequest.session.get(SessionKeys.lateAppealReason),
-            whoPlannedToSubmit = userRequest.session.get(SessionKeys.whoPlannedToSubmitVATReturn),
-            causeOfLateSubmissionAgent = userRequest.session.get(SessionKeys.causeOfLateSubmissionAgent)
+            isClientResponsibleForSubmission = userRequest.session.get(SessionKeys.whoPlannedToSubmitVATReturn).map(_ == "client"),
+            isClientResponsibleForLateSubmission = userRequest.session.get(SessionKeys.causeOfLateSubmissionAgent).map(_ == "client")
           )
         )
       }
       case "crime" => {
         AppealSubmission(
-          submittedBy = if (userRequest.isAgent) "agent" else "client",
-          reasonableExcuse = reasonableExcuse,
-          honestyDeclaration = userRequest.session.get(SessionKeys.hasConfirmedDeclaration).get == "true",
+          sourceSystem = "MDTP",
+          taxRegime = "VAT",
+          customerReferenceNo = s"VRN${userRequest.vrn}",
+          dateOfAppeal = LocalDateTime.now(),
+          isLPP = !userRequest.session.get(SessionKeys.appealType).contains(PenaltyTypeEnum.Late_Submission.toString),
```

Comment:
Could this be replaced with:
```suggestion
         isLPP = userRequest.session.get(SessionKeys.appealType).contains(PenaltyTypeEnum.Late_Payment.toString),
```